app/database/models/recipesModel.js

```diff
@@ -12,4 +12,37 @@ async function getById(id){
     return data;
 }
 
-module.exports={getById};
+/**
+ * insert recipe as a creator
+ * @param recipe
+ * @param uid
+ * @returns {Promise<{insertStatus: string}>}
+ */
+async function insertAsCreator(recipe, uid){
+    const result = {insertStatus: 'failed'};
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 10 spaces but found 16.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8445077)
```suggestion
          .insert(saved_recipe)
```